quartz/plugins/transformers/paragraphtovideo.ts

```diff
@@ -0,0 +1,29 @@
+import { visit } from "unist-util-visit"
+import { QuartzTransformerPlugin } from "../types"
+
+export const ParagraphToVideo: QuartzTransformerPlugin = () => {
```

Comment:
This should be a part of `ofm.ts` rather than a separate extension and should be configurable via option. Additionally, the way you've written the plugin won't work as you wrote a `rehype` transformer for `markdownPlugins()` (which only works with `remark` transformers)

You'll need to rewrite it to work with https://github.com/syntax-tree/mdast